cypress.config.js

```diff
@@ -0,0 +1,12 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  env: {
+    API_URL: "http://localhost:3001"
+  },
+  e2e: {
+    baseUrl: "http://localhost:3000",
+    supportFile: false,
+    fixturesFolder: false,
+  },
+});
```

Comment:
É uma boa prática deixar uma linha em branco ao final de cada arquivo.

```suggestion
});

```